Libraries/IntegrationServiceLibrary.psm1

```diff
@@ -521,17 +518,13 @@ function Create-Controller{
 	# Check if the controller already exists.
 	#
 	$scsiCtrl = Get-VMScsiController -VMName $vmName -ComputerName $server
-	if ($scsiCtrl.Length -1 -ge $controllerID)
-	{
-	Write-LogInfo "SCSI controller already exists"
-	}
-	else
-	{
+	if ($scsiCtrl.Length -1 -ge $controllerID) {
```

Comment:
if (($scsiCtrl.Length - 1) -ge $controllerID)

Reply:
fixed in 05932f86e8b954c3b1a541ffc483c96ca1036710